www/js/metrics/footprintHelper.ts

```diff
@@ -47,21 +46,10 @@ export function getFootprintForMetrics(userMetrics, defaultIfMissing = 0) {
   let result = 0;
   for (let i in userMetrics) {
     let mode = userMetrics[i].key;
-    if (mode == 'ON_FOOT') {
-      mode = 'WALKING';
-    }
 
+    //either the mode is in our custom footprint or it is not
     if (mode in footprint) {
       result += footprint[mode] * mtokm(userMetrics[i].values);
-    } else if (mode == 'IN_VEHICLE') {
```

Comment:
We can return `IN_VEHICLE` if we use cleaned modes, before the infer modes step of the pipeline.
That is why we see `IN_VEHICLE` in the public dashboard. There will not be an `IN_VEHICLE` mode in the custom labels, but it can still be returned if we set the server to use the cleaned modes instead of the inferred modes or the user labels (`conf/analysis/debug.conf.json.sample, `analysis.result.section.key`)

I would prefer to restore this until we decide how to handle anergy/emission calculations and/or count every trip.

**But** if it is too complicated, I would be open to leaving it off, since the fallback to the labeled trips is typically the inferred sections.

Reply:
I can add this back in, but I'm not sure I fully understand when this case would come up to be able to simulate it for testing. 

When we build the footprint in `customMetricsHelper` the keys that are used are the keys from the labels, and this snippet aims to average the numbers for 6 labels that might or might not be in that list. I think to do something like this we would need to refactor a bit to have base modes in the footprint

Maybe when the footprint is extracted from the labels we could average all the modes with a "vehicle" base mode and average them together and store it for use in this case?

Should I work on adding something like that to this PR or would it be better to wait?

Reply:
@Abby-Wheelis just restore it. Let's decide what to do with it when we revisit the calculations. No tests (which means that test coverage will be a bit lower) but that's OK for now.